interface/forms/aftercare_plan/new.php

```diff
@@ -26,153 +28,193 @@
+                // need to prepend first setting with a comma ?>
+            });
+        });
+    </script>
 
 </head>
 <body class="body_top">
-<p><span class="forms-title"><?php echo xlt('AfterCare Planning'); ?></span></p>
-<br />
-<?php
-echo "<form method='post' name='my_form' " .
-  "action='$rootdir/forms/aftercare_plan/save.php?id=" . attr_url($formid) . "'>\n";
-?>
-<input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
-
-<table  border="0">
-<tr>
-<td align="left" class="forms" class="forms"><?php echo xlt('Client Name'); ?>:</td>
-        <td class="forms">
-            <label class="forms-data"> <?php if (is_numeric($pid)) {
-                $result = getPatientData($pid, "fname,lname,squad");
-                echo text($result['fname']) . " " . text($result['lname']);
-                                       }
-
-                                       $patient_name = ($result['fname']) . " " . ($result['lname']);
-                                        ?>
-   </label>
-   <input type="hidden" name="client_name" value="<?php echo attr($patient_name);?>">
-        </td>
-        <td align="left"  class="forms"><?php echo xlt('DOB'); ?>:</td>
-        <td class="forms">
-        <label class="forms-data"> <?php if (is_numeric($pid)) {
-            $result = getPatientData($pid, "*");
-            echo text($result['DOB']);
-                                   }
-
-                                   $dob = ($result['DOB']);
-                                    ?>
-   </label>
-     <input type="hidden" name="DOB" value="<?php echo attr($dob);?>">
-        </td>
-        </tr>
-<tr>
-
-
-  <td align="left" class="forms"><?php echo xlt('Admit Date'); ?>:</td>
-        <td class="forms">
-               <input type='text' size='10' class='datepicker' name='admit_date' id='admission_date' <?php echo attr($disabled); ?>;
-               value='<?php echo attr($obj["admit_date"]); ?>'
-               title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
-        </td>
-        <td align="left" class="forms"><?php echo xlt('Discharged'); ?>:</td>
-        <td class="forms">
-               <input type='text' size='10' class='datepicker' name='discharged' id='discharge_date' <?php echo attr($disabled); ?>;
-      value='<?php echo attr($obj["discharged"]); ?>'
-       title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
-        </td>
-    </tr>
-    <tr>
-        <td align="left colspan="3" style="padding-bottom:7px;"></td>
-    </tr>
-        <tr>
-
-        <td class="forms-subtitle" colspan="4"><B><?php echo xlt('Goal and Methods');?></B></td>
-
-    </tr>
-    <tr>
-        <td align="left colspan="3" style="padding-bottom:7px;"></td>
-    </tr>
-    <tr>
-
-        <td class="forms-subtitle" colspan="4"><B><?php echo xlt('Goal A');?>:</B>&nbsp;<?php echo xlt('Acute Intoxication/Withdrawal'); ?></td>
```

Comment:
What happens to users that were using the previous form? (ie. when they edit old forms or make new forms looks like there will be changes in labels)

Reply:
I can see everything breaking. This is now a totally different form and should be saved under a new name. Once I changed the field names then the old saved data will not pull into the fields. If the fields were reverted back to their previous names at least the fields would populate with the old data. The goals would still be blank because they were hard coded before.
That is why I was just trying to update the look and not go to far as to destroy the backwards compatibility of the form. at this point the backwards compatibility is destroyed.